assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -656,3 +656,9 @@ che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
 
 # Docker image for the Che async storage
 che.infra.kubernetes.async.storage.image=quay.io/eclipse/che-workspace-data-sync-storage:latest
+
+# The timeout for shutdown Asynchronous Storage Pod after stopping last used workspace.
```

Comment:
stylistic suggestion:

```suggestion
# The timeout for the Asynchronous Storage Pod shutdown after stopping the last used workspace.
```

Reply:
also, let's clarify how to disable it ? set to 0?

Reply:
good catch, I will add such checking